### ISSUE TITLE
Tolerations aren't configurable between operator and daemonsets

### ISSUE THREAD

**Issue by bushong1** [aws-observability/helm-charts#48]
I'm trying to deploy the Operator on Fargate, which has a fargate taint, but obviously the DaemonSets can't run on Fargate notes.  To support this, the chart needs to be updated to handle different tolerations between operator and daemonsets

**Comment by Chili-Man** [comment]
Agreed, I'm running into this issue as well. There should be separate tolerations for the agent and operator.

**Comment by sky333999** [comment]
Root tolerations can now be configured and component level tolerations will soon be customizable once #154 is merged. Please follow https://github.com/aws/containers-roadmap/issues/2515 for updates on the same.
